installer.sh

```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env bash
 
+# check if we are 'root' user.
+(( EUID != 0 )) && echo Must be root to run this script. Exiting. &&
+    exit 1
+
 # get system unit directory
 DEST=$(pkg-config systemd --variable=systemdsystemunitdir)
+CONFDIR=/etc/g910-gkeys
+FILES=files.txt
+
+python3 setup.py install --record "$FILES"
 
-python3 setup.py install --record files.txt
+# systemd service file
+echo "$DEST"/g910-gkeys.service >> "$FILES"
 cp g910-gkeys.service "$DEST"/g910-gkeys.service
-systemctl daemon-reload
+
+# configuration file - will not overwrite existing files.
+[[ -d "$CONFDIR" ]] ||  mkdir "$CONFDIR"
+rsync -a -v --ignore-existing config/* "$CONFDIR"
```

Comment:
Except nothing is created in /etc/g910-gkeys when I do not do it in install script.
Beside that, even if it worked, apparently create_config.py renames existing config, and replaces it, which is very intrusive.

Reply:
Create config creates a configuration if user requests it with the flag --create-config. Config is also created if there is no config file in /etc/g910-gkeys/. 
Reasoning behind it is the same as other *nix programs, where if you delete the programs config, the program can still be run with default config. In the case of copying and removing create config, driver would crash if there is no config. User would have to go to github or old install dir and copy config files from there or install the whole driver.
Create config renames existing config only when --create-config is called, because the user specifically requests for a creation of default config.